lapis2-docs/src/components/ConfigGenerator/MetadataWizard.tsx

```diff
@@ -105,7 +105,7 @@ export function MetadataEditModal({ index, metadata }: { index: number; metadata
 
     const [metadataType, setMetadataType] = useState<MetadataType>(metadata.type);
     const [metadataName, setMetadataName] = useState(metadata.name);
-    const [generateIndex, setGenerateIndex] = useState(metadata.generateIndex);
+    const [generateIndex, setGenerateIndex] = useState(!!metadata.generateIndex);
```

Comment:
Shouldn't we also change the type of `generateIndex`? Or do we want to have explicitly in the config `generateIndex: false`. Then we would need to adapt this here and change the `handleSetMetadataType` accordingly

Reply:
I checked in SILO and the code looks like it doesn't care whether `generateIndex` is present or not. So I would go for this simple implementation here (which always adds `generateIndex`)